src/documents/management/commands/document_exporter.py

```diff
@@ -165,6 +166,11 @@ def add_arguments(self, parser):
             help="If set, the progress bar will not be shown",
         )
 
+        parser.add_argument(
+            "--passphrase",
+            help="If provided, is used to encrypt mail account passwords in the export",
```

Comment:
```suggestion
            help="If provided, is used to encrypt sensitive data in the export",
```

maybe just making this more generic for the future?